docs/dev/libvirt-howto.md

```diff
@@ -244,16 +244,13 @@ ssh core@$MASTER_IP
 
 ### Inspect the cluster with kubectl
 
-You'll need a `kubectl` binary on your path.
-If you launched the cluster with `openshift-install --dir "${DIR}" cluster`, you can use:
+You'll need a `kubectl` binary on your path and [the kubeconfig from your `cluster` call](../../README.md#kubeconfig).
```

Comment:
openshift usually recommends `oc`

Reply:
> openshift usually recommends `oc`

Orthogonal change to existing code ;).  File a follow-up PR?